### PR TITLE
Fix error with updating info on me/update page

### PR DESCRIPTION
**Pull request by sukhada**
PR to deploy hotfix for [this issue](https://github.com/350org/action-tools/issues/358)

### CODE REVIEW

File: user_update.html [hunk 1]
```diff
@@ -57,9 +57,23 @@ <h3 class="section-header meta margin-bottom-large">{{ user.name }}</h3>
 					{% case 'email' %}
 					{% if user.stats.donated %}
 						<p class="donor-email-label">We noticed you have previously donated. To update your email address ({{ field.value }}), please contact us at <a href="mailto:donations@350.org">donations@350.org</a> so we can make sure your donation records are moved to your new address.</p>
+						<input type="hidden" name="email" value="{{ field.value }}" maxlength="255" id="id_email" readonly>
```
Comment (cjmabry) [inline user_update.html:60]: Is this missing the updates made in the event create pages here or are those already in master? https://github.com/350org/ak_intl_v3/pull/470/files

**Review by cjmabry — COMMENTED** [review]
One question!

File: user_update.html (hunk 1 above)
Comment (sukhada) [reply user_update.html:60]: those were made in other commits (not authored by me) which I didn't want to include in this PR. it doesn't look like they're in master, so I'm not really sure where they came from 🤔

Comment (sukhada) [reply user_update.html:60]: oh nvm, they are included in master, they're just quite old: https://github.com/350org/ak_intl_v3/commit/38cce1b6a654a59eafc8268289e6251567a68784

Comment (cjmabry) [reply user_update.html:60]: Ah got it, makes sense!

**Review by cjmabry — APPROVED** [review]

**Pull request merged**